frontend/src/components/bucket/BucketList.vue

```diff
@@ -17,6 +27,23 @@ const closeInfo = () => {
   displayInfo.value = null;
 };
 
+const showBucketConfig = (isUpdate: boolean, bucket: Bucket) => {
+  if(isUpdate) {
+    bucketConfigHeader.value = BucketConfigConst.headerNewBucket;
+    bucketConfigTitle.value = bucket.bucketName;
+  } else {
+    bucketConfigHeader.value = BucketConfigConst.headerNewBucket;
+    bucketConfigTitle.value = BucketConfigConst.titleNewBucket;
+  }
+  isBucketConfigUpdate.value = isUpdate;
+  bucketToUpdate.value = bucket;
+  displayBucketConfig.value = true;
+};
```

Comment:
Can bucketConfigHeader be baked directly into the modal as there is no difference here? Then bucketConfigTitle can be set via ternary operator.